script.js

```diff
@@ -0,0 +1,26 @@
+const button2 = document.querySelector('.button');
+const xbutton = document.querySelector('.close');
+const module = document.querySelector('.modal');
+
+function revealContent() {
+    module.style.display = 'inline';
+}
+function hideContent() {
+    module.style.display = 'none';
+}
+
+document.addEventListener("click", function (e) {
```

Comment:
Instead of 1 addEventListener use separate event listeners for the functions.

Reply:
added to each element